lib/mayaUsd/ufe/UsdUndoCreateGroupCommand.cpp

```diff
@@ -53,26 +74,59 @@ Ufe::SceneItem::Ptr UsdUndoCreateGroupCommand::group() const
 
 void UsdUndoCreateGroupCommand::undo()
 {
-	if (!fGroup) return;
+#if UFE_PREVIEW_VERSION_NUM < 2017	
```

Comment:
Yes, I had been planning to move things around to make it more clear.  This is not a large file so I'm thinking of having the entire class split into two ifdef's, so the entire class/functions for the new interface being in one and the old interface in the other.  Especially if I change the parent class as well I think it would look better if all functions were together.